recipes/gritql/recipe.yaml

```diff
@@ -0,0 +1,84 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  version: "0.1.0-alpha.1743007075"
```

Comment:
Is this a released version? Probably not?

Reply:
Also: thanks for the speedy review! A bit more background: I _really_ love their [Markdown-based "literate linting"](https://docs.grit.io/guides/testing#markdown-format) approach, which I could see being useful, longer term, for describing `conda-smithy` linter rules, and potentially v1-based migrations. I caveat with `v1`, as I doubt `tree-sitter-yaml-jinja-with-crazy-inline-python-comments` is going to be worth anyone's time.

Reply:
Hi! I'm from the Grit team - you're fine to use this version. We haven't hit 1.0 yet so we do keep alpha in the release name, but this is in fact the latest release.